spec/factories/projects.rb

```diff
@@ -1,5 +1,8 @@
 FactoryBot.define do
   factory :project do
     title { Faker::Company.name }
+    trait :locked do
```

Comment:
What do you think about adding another trait for archived projects?
You can then use the trait here: https://github.com/fastruby/points/pull/271/files#diff-b4cf077b77d7765559bc95291a6e424d712c59271a5d6bcc7eb79b2cd3726e87R396